merge.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+set CWD=%~dp0
+set BIN="%CWD%\DaS-PC-MPChan\bin\Release"
+set IL="C:\il-repack\ILRepack\bin\Release"
+set PATH=%IL%;%PATH%
+
+ilrepack /log:merge.log /verbose /out:DSCM.exe %BIN%\DSCM.exe %BIN%\CommonMark.dll
```

Comment:
Does this merge CommonMark into the DSCM exe? What is the point of this? DSCM should not need commonmark to run – the markdown is compiled to HTML when DSCM is compiled.